packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -106,23 +114,26 @@ contract L2OutputOracle is Ownable {
 
     /**
      * @notice Accepts an L2 outputRoot and the timestamp of the corresponding L2 block. The
-     * timestamp must be equal to the current value returned by `nextTimestamp()` in order to be
-     * accepted.
-     * This function may only be called by the Sequencer.
-     * @param _l2Output The L2 output of the checkpoint block.
-     * @param _l2timestamp The L2 block timestamp that resulted in _l2Output.
-     * @param _l1Blockhash A block hash which must be included in the current chain.
-     * @param _l1Blocknumber The block number with the specified block hash.
+     *         timestamp must be equal to the current value returned by `nextTimestamp()` in order
+     *         to be accepted.
+     *         This function may only be called by the Sequencer.
+     *
+     * @param _l2Output      The L2 output of the checkpoint block.
+     * @param _l2BlockNumber The L2 block number that resulted in _l2Output.
+     * @param _l1Blockhash   A block hash which must be included in the current chain.
+     * @param _l1BlockNumber The block number with the specified block hash.
      */
     function appendL2Output(
         bytes32 _l2Output,
-        uint256 _l2timestamp,
+        uint256 _l2BlockNumber,
         bytes32 _l1Blockhash,
-        uint256 _l1Blocknumber
+        uint256 _l1BlockNumber
     ) external payable onlyOwner {
```

Comment:
timestamp and block number were swapped in the argument order. The specs still define this contract interface in the old order. Can you update the specs definition? (`specs/proposals.md`)